src/functions.js

```
@@ -2173,10 +2392,109 @@ export function drawIcon(icon,size,shade,id,inject){
+                        break;
+                    case 1:
+                    case 4:
+                    case 8:
+                        color = 'orange';
+                        break;
+                    case 10:
+                        color = 'black';
+                        global.race.pet['pattern'] = 'solid';
+                        break;
+                    default:
+                        let colors = ['black','white','gray','orange','cream'];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)